atlp-rwanda/fan7-bn-backend
# #171301342 Host supplier and Travel administrator can add rooms

### What does this PR do?
Host-suppliers and Travel-administrators can add rooms to their approved accommodation
### Description of Task to be completed?
Check if it is Host-suppliers or Travel-administrators
Validate inputs
Test our endpoint with mocha and chai
Allow User to attach multiple images to their accommodations
All rooms are ready to be booked

### How should this be manually tested?
git clone this repo and check out to `host-supplier-add-rooms-171301342`
run `npm I` to install all the dependencies
run sequelize db:migrate:undo: all then npm run migrate to add the like schema
run the application with npm run dev
register accommodation in the system

login the user and user the token provided in subsequent operations
- Use Postman: send a POST request to `/api/accommodations/:accommodation_id/room`  
### Any background context you want to provide?
N/A
What are the relevant pivotal tracker stories?
#171301342
https://www.pivotaltracker.com/story/show/171301342
Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/37307172/74684004-45d8b200-51d3-11ea-9e04-842e469c3386.png)
![image](https://user-images.githubusercontent.com/37307172/74745535-b67fdd00-526c-11ea-9aeb-b5d8c8d15528.png)
